app/src/main/java/org/jellyfin/androidtv/util/InfoLayoutHelper.java

```diff
@@ -374,7 +375,13 @@ private static void addSeriesStatus(Context context, BaseItemDto item, LinearLay
 
     private static void addVideoCodecDetails(Context context, LinearLayout layout, MediaStream stream) {
         if (stream != null) {
-            if (stream.getCodec() != null && stream.getCodec().trim().length() > 0) {
+            if (stream.getVideoDoViTitle() != null && stream.getVideoDoViTitle().trim().length() > 0) {
+                addBlockText(context, layout, "VISION");
+                addSpacer(context, layout, "  ");
```

Comment:
I believe you can use videoRangeType to check for Dolby Vision, the value should be "DOVI".

Reply:
It only outputs DOVI for Dolby Vision profile 5, which is not backwards compatible with any other range. Otherwise, it will mention the backwards compatible range eg. HDR10 for profile 8.1.